deploy/crds/ocmagent.managed.openshift.io_managedfleetnotifications.yaml

```diff
@@ -0,0 +1,80 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.10.0
+  creationTimestamp: null
+  name: managedfleetnotifications.ocmagent.managed.openshift.io
+spec:
+  group: ocmagent.managed.openshift.io
+  names:
```

Comment:
Just an option additionally. ManagedFleetNotification is quite long and it will help to have a shorter name too in `CRD.spec.names.shortNames`. Something like `mfn`?